### PR TITLE
Add tests for increase coverage 

### PR DESCRIPTION
**Pull request by geraxe**
To 100% coverage 🚀

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -7,6 +7,10 @@ skip_missing_interpreters = True
 [testenv]
 deps =
     setuptools-scm  # to get cached setuptools-scm on "sdist-make" step
+    pytest
+    pytest-asyncio
+    pytest-cov
+    pytest-recording
```
Comment (atugushev) [inline tox.ini:13]: These deps should be listed in `testing` extras in `setup.cfg`.

**Review by atugushev — APPROVED** [review]
Wow, that's huge! One comment above, otherwise 👍🏻

File: tox.ini (hunk 1 above)
Comment (geraxe) [reply tox.ini:13]: If you will use tox from console like _tox -p all_ this modules needs in config.

Comment (atugushev) [reply tox.ini:13]: `tox` installs everything that listed in `extras=testing`. So it's better to keep all testing modules in `setup.cfg`.

Comment (geraxe) [reply tox.ini:13]: Strangely did not work for me but I have now double-checked and it seems like yes tox really uses extras.testing
I will remove this commit

Comment (atugushev) [reply tox.ini:13]: Looks like `venv` created by tox was cached. `tox -p all --recreate` might help.

**Pull request merged**